src/bitcoind.cpp

```diff
@@ -19,6 +19,9 @@
 #include "httpserver.h"
 #include "httprpc.h"
 #include "utilstrencodings.h"
+#ifdef ENABLE_WALLET
```

Comment:
Right, we need to move the wallet dependency up here if we want to remove the wallet dependency from `libbitcoin_server`.